README-FIPS.md

```diff
@@ -23,7 +23,7 @@ For example, on Unix the final command
 
     $ make install
 
-effectively executes the following install targets
+effectively executes the following installation targets
```

Comment:
Well, they are `install_*` targets, and `make install` is idiomatic, so whether they are `install` or `installation` targets could be considered preference, rather than correctness

Reply:
Reverted back that too